fidesctl/src/fidesctl/core/generate_dataset.py

```diff
@@ -92,8 +137,14 @@ def generate_dataset(connection_string: str, file_name: str) -> str:
     Given a database connection string, extract all tables/fields from it
     and write out a boilerplate dataset manifest.
     """
+    database_ingestion_functions = {
+        "postgresql": get_postgres_collections_and_fields,
+        "mysql": get_mysql_collections_and_fields,
+        "mssql": get_mssql_collections_and_fields,
+    }
+
     db_engine = get_db_engine(connection_string)
-    db_collections = get_db_collections_and_fields(db_engine)
+    db_collections = database_ingestion_functions[db_engine.dialect.name](db_engine)
```

Comment:
might be good to have an error message for db_engine.dialect.name not being one of the keys we define

Reply:
it will actually fail before this happens, the `get_db_engine` function will throw an error before it hits here. I looked at doing this here and stumbled upon that error first, it will complain that the dialect doesn't exist